README.md

```diff
@@ -271,6 +271,10 @@ Build:
 
     yarn run build
 
+Code formatting:
```

Comment:
Let's also add a link so folks know how to set this up.

How about, "We use [prettier](link) for code formatting:"